### PR TITLE
Add some typos found in a large documentation repo

### LINKED ISSUE #3000: Misspellings found in a large documentation repo
```
adminsitrator->administrator
compatbile->compatible
endpiont->endpoint
histgram->histogram
improvemnt->improvement
improvemnts->improvements
latitute->latitude
specifc->specific
```

In addition:

```
vor->for
```

If these look okay, I'll submit a PR to add these. :+1:

### PR DESCRIPTION
**Pull request by kurtmckee**
Closes #3000

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -1992,6 +1992,8 @@ adminitrative->administrative
 adminitratively->administratively
 adminitrator->administrator
 adminitrators->administrators
+adminsitrator->administrator
+adminsitrators->administrators
```
Comment (int-y1) [inline codespell_lib/data/dictionary.txt:1996]: The surrounding suggestions included `administer(ed/ing/s)`, `administration(s)`, and `administrative(ly)`. Please add these words.

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:1996]: Fixed.

**Review by DimitriPapadopoulos — APPROVED** [review]

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (kurtmckee) [reply codespell_lib/data/dictionary.txt:1996]: @int-y1 thank you for catching that, and @DimitriPapadopoulos thank you for fixing those!

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:1996]: You're welcome, thank you for all the improvements so far.

**Pull request merged**